modules/appeals_api/config/locales/en.yml

```diff
@@ -1,6 +1,7 @@
 en:
   appeals_api:
       errors:
+        caseflow_bad_gateway: "Received an unusable response from Caseflow."
```

Comment:
Where is this being used? I don't think this needs to be present at all if we end up using the common exception I _think_ this was trying to be used with anyway.

Reply:
Looking into it.